elements/pl-graph/pl-graph.mustache

```diff
@@ -1,19 +1,3 @@
-<script>
-$(function() {
-  window.PLGraphRender({
-    uuid: "{{uuid}}",
-    workerURL: "{{workerURL}}",
-    data: {{{ data }}},
-    engine: "{{engine}}",
-  })
-})
-</script>
-
-<div id="pl-graph-{{uuid}}" class="pl-graph">
-  <!-- pure css loading spinner, nice -->
-  <div class="spinner">
-    <div class="double-bounce1"></div>
-    <div class="double-bounce2"></div>
-  </div>
-  <span class="render-label">Rendering...</span>
+<div class="pl-graph">
```

Comment:
@nwalters512 now that this file is essentially one line, do you want to remove it and just return a string from the Python controller?

Reply:
That would be great!

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/7326/commits/32e116b39ee162594d09c12bbf9b9a69fbd6004a